Fix some bugs discovered after testing

ItsDrike opened the pull request.

ItsDrike committed:
Fix parameter order for authorized fucntion

ItsDrike committed:
Merge branch 'main' into itsdrike-dev-2

ItsDrike committed:
Add required argument to fetchrow function

MaxTCodes commented on rickchurch/auth.py:
```diff
@@ -126,7 +126,7 @@ async def add_user(user: dict, db_conn: asyncpg.Connection) -> str:
 
     async with db_conn.transaction():
-        user_data = await db_conn.fetchrow("SELECT * FROM users WHERE user_id = $1")
+        user_data = await db_conn.fetchrow("SELECT * FROM users WHERE user_id = $1", user_id)
```
Would this prevent SQL injection?

ItsDrike replied:
Yes, passing arguments with this function is safe, using f-strings or other forms of string concatenation wouldn't be, but this is indeed safe.

MaxTCodes replied:
👌🏾,  I was just checking! Good job!

ItsDrike committed:
Log error if we access_token isn't aviable
- This can happen if OAuth2 redirect fails for some reason
- If this were to happen, we include the response JSON to see the error

ItsDrike committed:
Use proper directory path
